src/main/java/com/faforever/client/remote/domain/GameLaunchMessage.java

```diff
@@ -1,50 +1,33 @@
 package com.faforever.client.remote.domain;
 
+import com.faforever.client.fa.relay.LobbyMode;
+import com.faforever.client.game.Faction;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.Setter;
+
 import java.util.List;
 
+@Getter
+@Setter
 public class GameLaunchMessage extends FafServerMessage {
 
+  /**
+   * Stores game launch arguments, like "/ratingcolor d8d8d8d8", "/numgames 236".
+   */
   private List<String> args;
   private int uid;
   private String mod;
   private String mapname;
+  @NonNull
+  private String name;
+  private int expectedPlayers;
+  private int team;
+  private int mapPosition;
+  private Faction faction;
+  private LobbyMode initMode;
 
```

Comment:
Can we add the `numgames` parameter as a dedicated field in this message too instead of passing it via `args`?

Reply:
I would say yes why not

Reply:
Shall we call it `num_games`?

Reply:
I decided not to do this for now